src/sdk/namespace_cache.js

```diff
@@ -157,18 +168,27 @@ class NamespaceCache {
     /////////////////
 
     async list_objects(params, object_sdk) {
-        // TODO listing from cache only for deevelopment
-        return this.namespace_nb.list_objects(params, object_sdk);
+        const get_from_cache = params.get_from_cache;
+        params = _.omit(params, 'get_from_cache');
+        if (get_from_cache) {
+            return this.namespace_nb.list_objects(params, object_sdk).then(res => {
+                _.forEach(res.objects, obj => {
+                    obj.create_time = obj.last_modified_time ? obj.last_modified_time : obj.create_time;
+                });
+                return res;
+            });
+        }
+        return this.namespace_hub.list_objects(params, object_sdk);
     }
 
     async list_uploads(params, object_sdk) {
-        // TODO listing from cache only for deevelopment
-        return this.namespace_nb.list_uploads(params, object_sdk);
+
+        return this.namespace_hub.list_uploads(params, object_sdk);
```

Comment:
How about we use the same testing flag `req.query.get_from_cache` to allow both modes?
CC @paulxuzhang